posts/2024/sdk_12_3_1_rc.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Titanium SDK 12.3.1.RC released'
+date: '2024-05-30'
+category: 'Release'
+author: 'Hans Knöchel'
+teaser: 'Several bug fixes related to previous releases'
+---
```

Comment:
Every blog post should now have an `image`. This helps us with social prescence. The image goes in the `public/images` directory. Example: https://raw.githubusercontent.com/tidev/tidev-www/main/posts/2024/cli_7_1_0_ga.md.

See https://github.com/tidev/tidev-www#blog-posts for more info. What I would do is go to https://socialbu.com/tools/generate-blog-image, enter in a prompt like `black background with silky maroon waves flowing in the background 640x320`, generate an image until you find one you like. Download it, open it in some image editor (I bet Preview would work), drop a Titanium logo on it (https://github.com/tidev/titanium-sdk/blob/master/.github/logo-titanium.png or https://github.com/tidev/titanium-sdk/blob/master/.github/logo-titanium.svg maybe?), and say a few something like "Titanium SDK 12.3.1 RC".

I know this is a pain. Someday I could build a tool to generate the image. This is just an RC, so it's ok if we skip it this post, but the GA really should have a nice 640x320 image.